### ISSUE TITLE
Probleme mit dem neuen Layout bei Genios

### ISSUE THREAD

**Issue by SilkeBritz** [stefanw/bibbot#377]
Genios hat ein neues Layout bekommen. Vermutlich kann BibBot dadurch nur noch die Startseite öffnen. Der Artikel wird nicht mehr gesucht.

**Comment by emtiu** [comment]
This is the view for a Bücherhallen Hamburg account:
![Screenshot 2023-11-24 at 13-52-42 GENIOS eBIB - Die Rechercheplattform für Stadtbibliotheken](https://github.com/stefanw/bibbot/assets/2159933/9fb7b4bd-aede-4557-9f8a-8e14031d28a0)

**Comment by SilkeBritz** [comment]
Bücherhallen HH hat derzeit technische Probleme:
_wir haben aktuell eine allgemeine Störung unserer Systeme, was auch Einfluss auf die Authentifizierung bei unseren externen Dienstleistern hat. Wir arbeiten bereits an der Behebung der Störung. Bitte haben Sie etwas Geduld und versuchen Sie den Login später erneut._

**Comment by niwi-hh** [comment]
In Hamburg geht der Login bei Genios wieder. Das ändert aber leider nichts daran, dass Bibbot aktuell nicht mehr funktioniert.

**Comment by emtiu** [comment]
> In Hamburg geht der Login bei Genios wieder. Das ändert aber leider nichts daran, dass Bibbot aktuell nicht mehr funktioniert.

Stimmt, das Bild ist noch dasselbe wie [hier](https://github.com/stefanw/bibbot/issues/377#issuecomment-1825636843).

**Comment by McCisie** [comment]
Leider geht auch über die Stadtbibliothek Stuttgart die GENIOS Suche und das Einbetten des gefundenen Textes in die Ursprungsseite nicht mehr (gleiches Bild wie [oben](https://github.com/stefanw/bibbot/issues/377#issuecomment-1823216274)).
Suche ich manuell z.B. nach dem Titel des Artikels, wird er weiterhin gefunden.

Ein Fix wäre nett :-)

**Comment by Buchhems** [comment]
Stadtbibliothek Darmstadt funktioniert auch nicht mehr seit dem neuen Design

**Comment by stefanw** [comment]
Buecherhallen, Stuttgart, Darmstadt sind in v0.32.1 für das neue Interface konfiguriert.
Ansonsten gehen hier zu viele unterschiedliche Issues durcheinander.
